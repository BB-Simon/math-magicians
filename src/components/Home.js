const Home = () => (
  <div className="home-page-container">
    <h1 className="title">Welcome to our page!</h1>
    <p className="description">
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem
      omnis harum incidunt error blanditiis, exercitationem provident doloremque odio
      rem modi minima, aperiam magni quas praesentium mollitia dolor ex repellendus!
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem
      omnis harum incidunt error blanditiis, exercitationem provident doloremque odio
      rem modi minima, aperiam magni quas praesentium mollitia dolor ex repellendus!
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem
      omnis harum incidunt error blanditiis, exercitationem provident doloremque.
    </p>
    <p className="description">
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem
      omnis harum incidunt error blanditiis, exercitationem provident doloremque odio
      rem modi minima, aperiam magni quas praesentium mollitia dolor ex repellendus!
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem
      omnis harum incidunt error blanditiis, exercitationem provident doloremque odio
      rem modi minima, aperiam magni quas praesentium mollitia dolor ex repellendus!
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem
      omnis harum incidunt error blanditiis, exercitationem provident doloremque.
    </p>
  </div>
);

export default Home;
