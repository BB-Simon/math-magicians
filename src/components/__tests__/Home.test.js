import TestRenderer from 'react-test-renderer';
import Home from '../Home';

describe('Jest Snapshot testing suite', () => {
  it('renders correctly', () => {
    const domTree = TestRenderer.create(<Home />).toJSON();
    expect(domTree).toMatchSnapshot();
  });
});
