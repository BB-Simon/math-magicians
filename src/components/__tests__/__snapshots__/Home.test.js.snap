// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite renders correctly 1`] = `
<div
  className="home-page-container"
>
  <h1
    className="title"
  >
    Welcome to our page!
  </h1>
  <p
    className="description"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem omnis harum incidunt error blanditiis, exercitationem provident doloremque odio rem modi minima, aperiam magni quas praesentium mollitia dolor ex repellendus! Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem omnis harum incidunt error blanditiis, exercitationem provident doloremque odio rem modi minima, aperiam magni quas praesentium mollitia dolor ex repellendus! Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem omnis harum incidunt error blanditiis, exercitationem provident doloremque.
  </p>
  <p
    className="description"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem omnis harum incidunt error blanditiis, exercitationem provident doloremque odio rem modi minima, aperiam magni quas praesentium mollitia dolor ex repellendus! Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem omnis harum incidunt error blanditiis, exercitationem provident doloremque odio rem modi minima, aperiam magni quas praesentium mollitia dolor ex repellendus! Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus dolorem omnis harum incidunt error blanditiis, exercitationem provident doloremque.
  </p>
</div>
`;
